integration-tests/gradle/src/main/resources/build-configuration/gradle.properties

```diff
@@ -0,0 +1,6 @@
+quarkusPlatformArtifactId=quarkus-bom
+quarkusPlatformGroupId=io.quarkus
+quarkusPlatformVersion=999-SNAPSHOT
```

Comment:
Please do not use 999-SNAPSHOT (or any real Quarkus release) versions. We've run into issues before when globally changing versions when releasing.
You could see how we pass/complete current Quarkus version in other tests.

Reply:
I'll remove it as it's not required. I added it to try fixing a problem I had with the tests but it was unrelated and I forgot to remove it.